sonar-python-plugin/src/main/java/org/sonar/plugins/python/pylint/PylintRuleRepository.java

```diff
@@ -19,32 +19,232 @@
  */
 package org.sonar.plugins.python.pylint;
 
-import com.google.common.base.Charsets;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
 import org.sonar.api.server.rule.RulesDefinition;
 import org.sonar.api.server.rule.RulesDefinitionXmlLoader;
 import org.sonar.plugins.python.Python;
-import org.sonar.squidbridge.rules.SqaleXmlLoader;
 
 public class PylintRuleRepository implements RulesDefinition {
 
   public static final String REPOSITORY_NAME = "Pylint";
   public static final String REPOSITORY_KEY = REPOSITORY_NAME;
 
   private static final String RULES_FILE = "/org/sonar/plugins/python/pylint/rules.xml";
-  private static final String SQALE_FILE = "/com/sonar/sqale/python-model.xml";
   private final RulesDefinitionXmlLoader xmlLoader;
 
+  private static final Map<String, String> REMEDIATION_COST = initializeRemediationCost();
+
   public PylintRuleRepository(RulesDefinitionXmlLoader xmlLoader) {
     this.xmlLoader = xmlLoader;
   }
 
+  private static Map<String, String> initializeRemediationCost() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This method has 186 lines, which is greater than the 100 lines authorized. Split it into smaller methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS138)